quirks/password-rules.json

```diff
@@ -659,6 +659,9 @@
     "sfwater.org": {
         "password-rules": "minlength: 10; maxlength: 30; required: digit; allowed: lower, upper, [!@#$%*()_+^}{:;?.];"
     },
+    "store.nintendo.co.uk": {
+        "password-rules": "minlength: 8; maxlength: 20;"
+    },
```

Comment:
One last thing. Alphabetizing! One of our linters failed: https://github.com/apple/password-manager-resources/actions/runs/3572628412/jobs/6005732137

Reply:
Not doing too well with this PR am I! 😭

Reply:
You’re doing great!